data/cyp2c9_substrate_carbonmangels/transform.py

```diff
@@ -0,0 +1,158 @@
+import pandas as pd
+import yaml
+from tdc.single_pred import ADME
+
+def get_and_transform_data():
+    # get raw data
+    target_folder = 'CYP2C9_Substrate_CarbonMangels'
```

Comment:
did you actually add the split column to the new dataset?

Reply:
Sorry, I forget that. I will take a look again on the code of split.